pkg/cmd/browse/browse.go

```diff
@@ -59,8 +59,8 @@ func NewCmdBrowse(f *cmdutil.Factory, runF func(*BrowseOptions) error) *cobra.Co
 	}
 
 	cmd := &cobra.Command{
-		Long:  "Open the GitHub repository in the web browser.",
-		Short: "Open the repository in the browser",
+		Long:  "Open the GitHub repository, path inside it, commit, issue or pull request in the web browser.",
+		Short: "Open the repository, path inside it, issue or PR in the browser",
```

Comment:
**Is there any reason we shouldn't bring this more in line with other commands with a longer-`heredoc` `Long`?**

Like many of our other commands, I think the `Long` field should be expanded into the same multi line format we use elsewhere, allowing more content to be captured in a readable way.

For example:

https://github.com/cli/cli/blob/d60e5ffba335281439d95749abb6f4c3b94fabdc/pkg/cmd/api/api.go#L72-L104

**I'm not sold on the actual content below, but I think this would be easier to understand if it was reformatted**

```suggestion
		Short: "Open repositories, issues, pull requests, and more in the browser",
		Long:  heredoc.Doc(`
			Transition from the terminal to the web browser to view and interact with:

			- issues
			- pull requests
			- repository content
			- repository profile
			- repository settings
		`),
```